src/scripts/http.js

```diff
@@ -66,6 +66,7 @@ export class Http {
                 req.setRequestHeader(key, self._headers[key]);
               }
               if (bearer && bearer.token) {
+                localStorage.setItem("request-token", bearer.token);
```

Comment:
This may be a security issue https://stormpath.com/blog/where-to-store-your-jwts-cookies-vs-html5-web-storage

Reply:
Yeah, i know, but i need discussion here with you. Seems, when CORS is happen access token not stored in cookies in total, so, every request will be 401, and then request to auth in SSO, and then request to resource again. Imagine you have 50 image tags on front, fetching all of them without caching the key causes 150 requests :( You can check that in example